wolfcrypt/test/test.c

```diff
@@ -10111,6 +10111,14 @@ int rsa_no_pad_test(void)
         || out == NULL || plain == NULL
     #endif
     ) {
+        /* In non-async case safe to assume tmp failed to malloc, out
+         * and plain are both stack, no need to check. In the event of
+         * async any one may have triggered entry, check all three */
+    #ifdef WOLFSSL_ASYNC_CRYPT
+        if (tmp != NULL) XFREE(tmp, HEAP_HINT, DYNAMIC_TYPE_TMP_BUFFER);
```

Comment:
XFREE requires a NULL check already. FREE_VAR also already does a NULL check and is a macro supported with and without async. You should be able to have this section just be `ERROR_OUT(-6900, exit_rsa_nopadding);`